Installing-and-Configuring/OS/index.md

```diff
@@ -77,9 +83,11 @@ This feature may be turned off during configuration of heads.  The following wil
   CONFIG_TPM_NO_LUKS_DISK_UNLOCK=y
 
 
-### Using only a LUKS passphrase
+### Using the LUKS Disk Recovery Key / Using only a LUKS passphrase
+
+In this model heads is not involved in processes related to the decryption of data on disk. The OS manages keys and access to the LUKS containers by asking the user for a passphrase on boot.  
```

Comment:
In this model heads is not involved in processes related to the decryption of data on disk. The OS manages keys and access to the LUKS containers by asking the user for a passphrase on boot.

Technical clarification here, as clarified in counter proposition to your text block above. Pick, but do not modify. I feel we have been through this multiple times which is why i'm now linking in code. I do not see the point in documenting wrongly what is in code. So let's try by another angle here again:

The Disk unlock Key, per se,[ is generated under Heads from random number generator to create a 128 characters key](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-seal-key#L54-L61) at the moment of [renewing/changing Disk Unlock Key and Disk Unlock Key passphrase](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/kexec-save-default#L48-L114) upon [requirement of defining a new boot default when /boot integrity changed](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/initrd/bin/gui-init#L91-L97), which changes happens at core OS upgrades (under Qubes, when updating dom0 hyperviser boot related files, being Xen, kernel, initrd and grub config. Other Operating Systems could change those files at any automatic upgrade, so inspecting the upgrade log and rebooting accordingly is advised.)

This randomized key is used to generate LUKS slot1 key, injecting it by having the right Disk Recovery Key passphrase to unlock the LUKS container.
Then, that key is sealed under TPM nv memory